packages/palette/src/elements/Input/Input.tsx

```diff
@@ -208,9 +208,9 @@ const StyledLabel = styled.label<StyledInputProps>`
   left: 5px;
   padding: 0 5px;
   background-color: ${themeGet("colors.white100")};
-  transform: translate(0, -50%) scale(1);
+  transform: translateY(-50%);
   transition: 0.25s cubic-bezier(0.64, 0.05, 0.36, 1);
-  transition-property: color, transform, padding;
+  transition-property: color, transform, padding, font-size;
```

Comment:
Changed these to fix this issue from showing up

### Before
<img width="870" alt="image" src="https://user-images.githubusercontent.com/20655703/220163440-411b71ac-e41e-494b-b561-1b44b90f254a.png">

After: 
<img width="867" alt="image" src="https://user-images.githubusercontent.com/20655703/220163468-1f72d735-b2ea-42b5-87b9-c7fb9c90e575.png">